tests/Aspire.Hosting.Azure.Tests/Snapshots/AzureBicepResourceTests.AddAzureStorageViaPublishMode.verified.bicep

```diff
@@ -28,6 +28,10 @@ resource blobs 'Microsoft.Storage/storageAccounts/blobServices@2024-01-01' = {
   parent: storage
 }
 
+resource queues 'Microsoft.Storage/storageAccounts/queueServices@2024-01-01' = {
```

Comment:
Do we have to make this change? If you didn't call `AddQueues`, is there a reason to add this resource to the bicep?

Reply:
We're doing the same for the blobs. But it's a fair question.